canister/src/blocktree.rs

```diff
@@ -185,7 +187,23 @@ fn get_chain_with_tip_reverse<'a, 'b>(
     None
 }
 
+fn get_max_weight(tree: &BlockTree, network: BitcoinNetwork) -> u128 {
```

Comment:
```suggestion
fn difficulty_based_depth(tree: &BlockTree, network: BitcoinNetwork) -> u128 {
```
The terminology in the [wiki](https://wiki.internetcomputer.org/wiki/Bitcoin_integration) has been updated to not use the term "weight" anymore, as that conflicts with another concept in Bitcoin.